portfolio/src/main/webapp/index.html

```diff
@@ -23,7 +23,9 @@ <h1>a page about me</h1>
                 <br>
                 <a href="https://github.com/biancamacias/" target="_blank">Github</a>
                 <br>
-                <p id="quote-container" onclick="promptRandomQuote()">daily quote generator!</p>
+                <div id="quoteDisplayer">
```

Comment:
I think [all code should be lower case](https://google.github.io/styleguide/htmlcssguide.html#Capitalization) including class names. So i think your original id naming convention was better.